docker-compose.override.example.yaml

```diff
@@ -0,0 +1,28 @@
+# By default, Compose reads two files, a docker-compose.yml and an
+# optional docker-compose.override.yml file. By convention,
+# the docker-compose.yml contains your base configuration. The
+# override file, as its name implies, can contain configuration
+# overrides for existing services or entirely new services.
+# See https://docs.docker.com/compose/extends/
+#
+# Usage: copy this file, name it ".docker-compose.override.yml" and
+# set your own values.
+
+services:
+  # Add "web" prefix to distinguish it from the "dev" service on the back-end.
+  web-dev:
+    build:
+      context: .
+      args:
+        NGINX_CONF: nginx.local.conf
+      target: dev
+    environment:
+      NODE_ENV: development
+      # The API root URL, can point to localhost or a remote host.
+      # Should include trailing slash. Is just a "/" if the API has the same base URL.
```

Comment:
This last sentence is unclear for me. Do you mean something like this?

Relative API root URLs are allowed. A "/" is used when the API is hosted on the root ("/") of the same domain as this service.

And am I correct that the API can then also be hosted on "/api" etc?

If the base url of the frontend is "/frontend" and the app_api_root is "/api". Where is the api then hosted? If it is "/frontend/api" then I am confused by the variable naming. If it is still at "/api" then that makes sense to me

Reply:
I'll see if I can make the comments easier to understand.